README.md

```diff
@@ -90,13 +127,24 @@ The 1Password utility is a light wrapper of the [1Password Connect Python SDK](h
 
 The Slack operator utility makes use of the integration between the Airflow and a Slack app webhook. The purpose of the utility is to add Slack notifications to DAGs using the [callback](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/callbacks.html#callback-types) parameters. Failure, critical failure, and success notifications are implemented.
 
+## Useful Commands
+
+- 🐚 get a shell on a worker, for example
+
+```
+docker exec -it airflow-airflow-worker-1 bash
+```
+
+- ⛔ Stop all containers and execute this to reset your local database.
+  - Do not run in production unless you feel really great about your backups.
+  - This will reset the history of your dag runs and switch states.
+
+```
+docker compose down --volumes --remove-orphans
+```
+
 ## Ideas
```

Comment:
Just caught this after scanning through. Should we capture these in issues and remove?

Reply:
I think this is actually valuable to keep around for local stack users. We'd obviously never do it in production, but it can be nice to give yourself a tabula rasa locally. I'm game either way though.